lib/index.js

```diff
@@ -164,3 +167,45 @@ const findNodesForFilter = () => {
 const removeFoodRow = () => {
   event.target.closest('article').remove()
 }
+
+const calorieSorter = (originalOrder) => {
+  var count = 0
+
+  $('strong.calorie-sort').on('click', event => {
+    if (originalOrder.length === 0) {
+      var order = $('div.all-foods-list article.food-item-row')
+      order.each(index => {
+        originalOrder.push(order[index])
+      })
+    } else if (originalOrder.length != $('div.all-foods-list article.food-item-row').length) {
```

Comment:
This piece resets the original order if a user adds a new food so that the new food is included in the original order.